server/sonar-server/src/test/java/org/sonar/server/issue/ws/SearchActionTest.java

```diff
@@ -151,6 +151,19 @@ public void test_definition() {
       "Portfolios are not supported. If this parameter is set, 'projects' must not be set.");
   }
 
+  //SONAR-10217
+  @Test
+  public void test_with_incorrect_branch() throws Exception {
```

Comment:
remark: prefix with `test` is not needed. I'd name the test: `empty_search_with_unknown_branch`